Resources/public/js/models/ez-contentmodel.js

```diff
@@ -212,6 +212,10 @@ YUI.add('ez-contentmodel', function (Y) {
                 return this.get('relations');
             }
 
+            if (type === "ATTRIBUTE") {
```

Comment:
actually you should check if a field definition identifier was provided because this implies `ATTRIBUTE` type and this would not break the case `content.relations('ATTRIBUTE')`